jobs/JGLOBAL_WAVE_POST_SBS

```diff
@@ -19,6 +19,14 @@ for out_dir in "${COMOUT_WAVE_PREP}" "${COMOUT_WAVE_GRID}"; do
     if [[ ! -d "${out_dir}" ]]; then mkdir -p "${out_dir}"; fi
 done
 
+for grid in "0p25" "0p50" "1p00"; do
```

Comment:
@AntonMFernando-NOAA  I know this is a work in progress  - but I was looking at this and I'm curious how these grids here were chosen?  Especially as the description of the PR seems to imply that we'll use directory names such as global.0p16 or arctic.9km?

Reply:
> @AntonMFernando-NOAA I know this is a work in progress - but I was looking at this and I'm curious how these grids here were chosen? Especially as the description of the PR seems to imply that we'll use directory names such as global.0p16 or arctic.9km?

@JessicaMeixner-NOAA Yes. I know. I was assigned this on Monday and I did some editing to start the PR. This is just for testing and get the idea what files need to be changed. Have to talk to Walter about this soon one I get the hang of it. Please ignore the first few commits I have made. Once I am ready will let you know for review.

Reply:
Sounds great @AntonMFernando-NOAA ! Let me know if there's anything I can help point out or explain for waves!

Reply:
@JessicaMeixner-NOAA Will do. Thank you .